### ISSUE TITLE
Scrubbing broke

### ISSUE THREAD

**Issue by code8buster** [gerbera/gerbera#2804]
Between 1.12.0  `a40aa4d5` and 1.12.1 `b6370820` scrubbing broke for me. I'm compiling with the following options.

```
     -DCMAKE_BUILD_TYPE=Release \\                                                                                                                                                                                                                         
     -DCMAKE_INSTALL_PREFIX=/usr \\                                                                                                                                                                                                                        
     -DCXX_FILESYSTEM_HAVE_FS=ON \\                                                                                                                                                                                                                        
     -DEXIF_ROOT_DIR=/usr \\                                                                                                                                                                                                                               
     -DSTATIC_LIBUPNP=OFF \\                                                                                                                                                                                                                               
     -DWITH_AVCODEC=ON \\                                                                                                                                                                                                                                  
     -DWITH_CURL=ON \\                                                                                                                                                                                                                                     
     -DWITH_DEBUG=ON \\                                                                                                                                                                                                                                    
     -DWITH_EXIF=ON \\                                                                                                                                                                                                                                     
     -DWITH_EXIV2=ON \\                                                                                                                                                                                                                                    
     -DWITH_FFMPEGTHUMBNAILER=ON \\                                                                                                                                                                                                                        
     -DWITH_INOTIFY=ON \\                                                                                                                                                                                                                                  
     -DWITH_JS=ON \\                                                                                                                                                                                                                                       
     -DWITH_LASTFM=OFF \\                                                                                                                                                                                                                                  
     -DWITH_MAGIC=ON \\                                                                                                                                                                                                                                    
     -DWITH_MATROSKA=ON \\                                                                                                                                                                                                                                 
     -DWITH_MYSQL=OFF \\                                                                                                                                                                                                                                   
     -DWITH_NPUPNP=ON \\                                                                                                                                                                                                                                   
     -DWITH_SYSTEMD=OFF \\                                                                                                                                                                                                                                 
     -DWITH_TAGLIB=ON \\                                                                                                                                                                                                                                   
     -DWITH_TESTS=ON
```
Haven't sat down and done a full bisect. When trying to scrub through a video, the TV reports the file cannot be recognized, and can no longer play the file.

**Comment by code8buster** [comment]
I attempted to bisect this, but that yielded no change. The difference was I'd compiled with standard libupnp for 1.12.0

If you want to retain compatibility with libnpupnp >=5.0.0 some changes may be required. I was unable to get any debug output relating to this lib during my testing.

Compiling 1.12.1.x with libupnp and -DWITH_NPUPNP=OFF results in gerbera able to play every file, resume files, and scrub media files.

**Comment by whyman** [comment]
Interesting, generally we only test with libupnp.

Did libnpupnp 5 build ok? It just had side effects?

**Comment by code8buster** [comment]
Yeah it builds without complaint. The side effects are as described, no scrub and certain files won't play (tended to be 4K UHD media)

**Comment by whyman** [comment]
Interesting, I'm sure if you had debug logs @medoc92 might be interested

**Comment by medoc92** [comment]
I'd be glad to take a look, but I'd need a test case (and details about the platforms).

**Comment by medoc92** [comment]
Actually, I've found a bug in how range requests were processed (data ok but not status or header). This did not bother most audio renderers, but prevented the current vlc, for example, from seeking (I think that it used to work). Anyway, I released npupnp 5.0.1, @code8buster could you please give it a try ?

**Comment by code8buster** [comment]
Works perfectly, thank you.
